gitignore

```diff
@@ -1,38 +0,0 @@
-# See https://help.github.com/articles/ignoring-files for more about ignoring files.
```

Comment:
Let `rails new` manage `.gitignore` automatically. We don't need to provide our own.